### PR TITLE
Small fixes to test and imports

### PR DESCRIPTION
**Pull request by kwek20**
residual from the `use_excess_if_low` pr

### CODE REVIEW

File: sdk/tests/wallet/output_preparation.rs [hunk 1]
```diff
@@ -194,7 +194,7 @@ async fn output_preparation() -> Result<()> {
     // required storage deposit
     assert_eq!(output.unlock_conditions().unwrap().len(), 2);
     // metadata feature
```
Comment (Thoralf-M) [inline sdk/tests/wallet/output_preparation.rs]:
```suggestion
    // metadata and tag features
```

File: sdk/tests/wallet/output_preparation.rs [hunk 2]
```diff
@@ -166,7 +166,7 @@ async fn output_preparation() -> Result<()> {
...
```
Comment (Thoralf-M) [inline sdk/tests/wallet/output_preparation.rs]:
```suggestion
    // metadata and tag features
```

File: sdk/tests/wallet/output_preparation.rs [hunk 3]
```diff
@@ -141,7 +141,7 @@ async fn output_preparation() -> Result<()> {
     // output amount -1
     assert_eq!(storage_deposit_return.amount(), 48199);
     // metadata feature
```
Comment (Thoralf-M) [inline sdk/tests/wallet/output_preparation.rs]:
```suggestion
    // metadata and tag features
```

File: sdk/tests/wallet/output_preparation.rs [hunk 4]
```diff
@@ -112,7 +112,7 @@ async fn output_preparation() -> Result<()> {
     // only address condition
     assert_eq!(output.unlock_conditions().unwrap().len(), 1);
     // metadata feature
```
Comment (Thoralf-M) [inline sdk/tests/wallet/output_preparation.rs]:
```suggestion
    // metadata and tag features
```

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
tag feature requires additional storage deposit, so test need to be updated

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
conflict

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**